CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Snyk Vulnerability Scanner Changelog
 
+## [2.4.1]
+### Fixed
+- Fix ClassCastException exception after plugin update (Many Thanks everyone who did report it!)
```

Comment:
@ArtsiomCh , how about `Fix ClassCastException when updating the plugin without rebooting IDE`?
So we explain in changelog the condition when it happens.